### PR TITLE
[tests-only] expected failures of favourites tests

### PR DESCRIPTION
**Pull request by individual-it**
run the favourites tests added/changed in https://github.com/owncloud/core/pull/38047

### CODE REVIEW

File: ocis/tests/acceptance/expected-failures-on-OCIS-storage.txt [hunk 1]
```diff
@@ -1632,3 +1632,47 @@ apiShareOperationsToShares/changingFilesShare.feature:40
...
+#
+apiVersions/fileVersionsSharingToShares.feature:179
+#
+# not possible to move file into a received folder https://github.com/owncloud/ocis/issues/764
+#
+apiVersions/fileVersionsSharingToShares.feature:222
+apiVersions/fileVersionsSharingToShares.feature:223
+apiVersions/fileVersionsSharingToShares.feature:224
+apiVersions/fileVersionsSharingToShares.feature:225
+The version number of a file is incorrect because of the incorrect number of <d:getetag> and <d:getlastmodified> element#
```
Comment (phil-davis) [inline ocis/tests/acceptance/expected-failures-on-OCIS-storage.txt:1670]: Random crud :)

File: ocis/tests/acceptance/expected-failures-on-OWNCLOUD-storage.txt [hunk 2]
```diff
@@ -1539,3 +1539,40 @@ apiShareOperationsToShares/changingFilesShare.feature:40
...
+#
+apiVersions/fileVersionsSharingToShares.feature:179
+#
+# not possible to move file into a received folder https://github.com/owncloud/ocis/issues/764
+#
+apiVersions/fileVersionsSharingToShares.feature:222
+apiVersions/fileVersionsSharingToShares.feature:223
+apiVersions/fileVersionsSharingToShares.feature:224
+apiVersions/fileVersionsSharingToShares.feature:225
+The version number of a file is incorrect because of the incorrect number of <d:getetag> and <d:getlastmodified> element#
```
Comment (phil-davis) [inline ocis/tests/acceptance/expected-failures-on-OWNCLOUD-storage.txt:1566]: Random crud

File: ocis/tests/acceptance/expected-failures-on-OCIS-storage.txt (hunk 1 above)
Comment (individual-it) [reply ocis/tests/acceptance/expected-failures-on-OCIS-storage.txt]: fixed

File: ocis/tests/acceptance/expected-failures-on-OWNCLOUD-storage.txt (hunk 2 above)
Comment (individual-it) [reply ocis/tests/acceptance/expected-failures-on-OWNCLOUD-storage.txt]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**